travis/test_server.py

```diff
@@ -350,12 +351,12 @@ def main(argv=None):
 
     if odoo_unittest:
         to_test_list = tested_addons_list
-        cmd_odoo_install = [
-            "%s/%s" % (server_path, script_name),
-            "-d", database,
-            "--stop-after-init",
-            "--log-level=warn",
-        ] + install_options + ["--init", None] + server_options
+        cmd_odoo_install = ["%s/openerp-server" % server_path,
+                            "-d", database,
+                            "--stop-after-init",
+                            "--log-level=warn",
+                            ] + install_options + ["--init", None] +\
+                            server_options
```

Comment:
Indeed this is a revert of my initial solution, followed by a more elegant and pythonic solution in the second commit.